ios/Framework/Core/Auth/SignUpScreenlet/Connectors/UpdateCurrentUserLiferayConnector.swift

```diff
@@ -48,11 +48,11 @@ public class UpdateCurrentUserLiferayConnector: ServerConnector {
 	}
 
 	public func attributeAsString(key: String) -> String {
-		return SessionContext.currentContext?.userAttribute(key) as! String
```

Comment:
Same here: add `userAttribute(key)` method.
Keep in mind the [Liskov Substitution Principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle), or as a simple rule of thumb: don't chain more than one period in an expression (don't access one object internals, just ask the object to do something)